bin/cssbeautify

```diff
@@ -56,14 +56,23 @@ process.argv.splice(2).forEach(function (entry) {
         console.log('CSS Beautify version 0.3.0');
         console.log();
         process.exit(0);
+    } else if (entry === '-o' || entry === '--output') {
+        // Set the output file
+        ofile = arr[index+1];
     } else if (entry.slice(0, 2) === '--') {
         console.log('Error: unknown option ' + entry + '.');
         process.exit(1);
     } else if (typeof fname === 'string') {
-        console.log('Error: more than one input file.');
-        process.exit(1);
+		var prevArgv = arr[index-1];
+        if (prevArgv !== '-o' && prevArgv !== '--output') {
+			console.log('Error: more than one input file.');
+			process.exit(1);
+		}
```

Comment:
Something wonky going on with formatting here.

Reply:
Formatted the code :)